huaweicloud/services/meeting/config.go

```diff
@@ -45,35 +45,51 @@ const (
 )
 
 func buildTokenEnvKey(d *schema.ResourceData) string {
+	var corpId, userId string
+	if val, ok := d.GetOk("corp_id"); ok {
+		corpId = val.(string)
+	}
+	if val, ok := d.GetOk("user_id"); ok {
```

Comment:
if user_id does not exist,  will append nil in line 55.
maybe using the array. join() is better.

Reply:
not nil, is empty string